packages/tools/src/types/ContourAnnotation.ts

```diff
@@ -1,13 +1,19 @@
 import type { Types } from '@cornerstonejs/core';
 import { Annotation } from './AnnotationTypes';
+// Import the type so it isn't recursive imports
+import type { PointsArray } from '../utilities/contours/PointsArray';
 
 export type ContourAnnotationData = {
   data: {
     contour: {
       polyline: Types.Point3[];
       closed: boolean;
     };
+    handles: {
+      interpolationSources?: PointsArray<Types.Point3>[];
```

Comment:
can you add a comment what is this interoplationSources

Reply:
Once again, I don't expect any mention of interpolation in ROI tools. However, if you take a look at the contourSegmentation counterparts, you will see that we have included them there.